tests/k8s/1-022_validate_notifications/03-create-app.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: |
+    set -e
+
+    cat << EOF | oc apply -f -
```

Comment:
Should probably use `kubectl` rather than `oc` since this isn't OpenShift specific.

Reply:
updated, thanks!